### ISSUE TITLE
Offline Multiplayer

### ISSUE THREAD

**Issue by TiPunch69** [raceintospace/raceintospace#346]
Just an idea: 
Is it possible to play against another human who has the game installed remotely?

**Comment by peyre** [comment]
I agree, Michael.  I'd dearly love to see PBEM back in the game.  Even better would be TCP/IP, but PBEM would be better.  There are Issues started for both of those.  (Unfortunately it's much easier to open an Issue than it is to implement it.)  You wouldn't be a programmer, would you by any chance?

**Comment by TiPunch69** [comment]
Again no, not really, sorry :-(
